x-pack/plugins/security/server/lib/authentication/providers/__tests__/saml.js

```diff
@@ -54,7 +55,7 @@ describe('SAMLAuthenticationProvider', () => {
       sinon.assert.calledWithExactly(
         callWithInternalUser,
         'shield.samlPrepare',
-        { body: { acs: `test-protocol://test-hostname:1234/test-base-path/api/security/v1/saml` } }
+        { body: { realm: `saml1` } }
```

Comment:
Yes, that's the plan! I'll squash and push all subsequent commits as one and remove the WIP label when it's ready for a final review, so that you don't get sporadically pinged :)

Reply:
Sounds good, feel free to push whenever, and I'll only take a look when you're ready for the actual review. Thanks for taking this on!